src/utils/paymentQRHelper.test.ts

```diff
@@ -0,0 +1,99 @@
+import { Transaction } from "../types";
+import {
+  findValueByKey,
+  updateTransactionsPaymentQRIdentifiers,
+} from "./paymentQRHelper";
+
+describe("tests for paymentQRHelper", () => {
```

Comment:
small nit, dont think need to group the tests for `findValueByKey` and `updateTransactionsPaymentQRIdentifiers` under the same describe block, feels a bit unnecessary haha. what do you think?

Reply:
Will separate them!